package.json

```diff
@@ -35,6 +35,23 @@
     "watch:peregrine": "cd packages/peregrine && npm run -s watch; cd - >/dev/null",
     "watch:venia": "cd packages/venia-concept && npm run -s watch; cd - >/dev/null"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "npm test"
```

Comment:
We're pretty steadfast on not having a `pre-commit` hook, please perform both `lint-staged` and `npm test` to `pre-push`.

Reply:
In this case it is not obvious what to do with lint-staged, because the idea of lint-staged command is to fix prettier and eslint issues automatically and add the changes to the commit. What should we do in case we decide to run lint-sraged on pre-push? I believe we need to create a separate commit for the changes. What will be the message of this commit?

Reply:
In general, we can just run our `npm run prettier`, `npm run lint` commands on prepush instead of lint-staged, but in this case we will remove all automation implemented in this pull request, because we will need to create commit with fixes manually.